src/webui/www/private/scripts/addtorrent.js

```diff
@@ -44,10 +44,10 @@ window.qBittorrent.AddTorrent ??= (() => {
     let source = "";
     let downloader = "";
 
-    const localPreferences = new window.qBittorrent.LocalPreferences.LocalPreferences();
+    const clientData = window.parent.qBittorrent.ClientData;
 
     const getCategories = () => {
-        const defaultCategory = localPreferences.get("add_torrent_default_category", "");
+        const defaultCategory = clientData.get("add_torrent_default_category") ?? "";
```

Comment:
IIRC you mentioned about 3rd party webui might also use `api/v2/clientdata`. And currently webui is setting a bunch of keys. Won't it cause collision with other webui?
One idea is to prefix these keys with some unique string such as `qbt_webui_`. This can be handled transparently within `window.parent.qBittorrent.ClientData` functions.

Reply:
> One idea is to prefix these keys with some unique string

Such idea was suggested from the very beginning: https://github.com/qbittorrent/qBittorrent/pull/23088#issuecomment-3179328101.

Reply:
> IIRC you mentioned about 3rd party webui might also use `api/v2/clientdata`. And currently webui is setting a bunch of keys. Won't it cause key collision with other webui?

I do not think third party clients should use this API. We can't stop them if they do, but it's not the intended use. Third party clients will be responsible for prefixing their keys - we don't have to prefix our own first party keys.

Reply:
>It's simply not the intended use case. They can use it if they want, we won't restrict it, 

Good luck dictating/enforcing this, it is public API after all.

Reply:
I've modified the `ClientData` class to add a static prefix to all keys when interacting with the API. The prefix is totally transparent to callers of `ClientData`.